libraries/core_libs/network/src/tarcap/packets_handlers/status_packet_handler.cpp

```diff
@@ -51,6 +52,17 @@ void StatusPacketHandler::process(const PacketData& packet_data, const std::shar
     auto const node_major_version = (*it++).toInt<unsigned>();
     auto const node_minor_version = (*it++).toInt<unsigned>();
     auto const node_patch_version = (*it++).toInt<unsigned>();
+    auto const node_history = (*it++).toInt<uint64_t>();
```

Comment:
if we are going to use "is_light_node" flag in config, I would use it also here + light_node_history